contrib/pg_tde/src/pg_tde_event_capture.c

```diff
@@ -301,6 +303,7 @@ reset_current_tde_create_event(void)
 	tdeCurrentCreateEvent.encryptMode = false;
 	tdeCurrentCreateEvent.baseTableOid = InvalidOid;
 	tdeCurrentCreateEvent.relation = NULL;
+	tdeCurrentCreateEvent.tid = 0;
```

Comment:
Shouldn't this be `InvalidFullTransactionId`?

Reply:
Changed, but I don't think it makes the code better.